app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
+| `_vis_opt_exp_{CAMPAIGNID}_split`             | This cookie is created when one of the variations is chosen for you, but you are not yet redirected to the variation page. It exists so that VWO can verify that the variation page corresponds to the chosen variation after landing on the page        | 100 days                                        |
+| `_vis_opt_s`                                  | This VWO cookie tracks sessions                                                                                                                                                                                                                          | 100 days                                        |
+| `_vis_opt_out`                                | This VWO cookie indicates that you should not be made part of any campaign                                                                                                                                                                               | 10 years                                        |
+| `_vwo_uuid`                                   | This VWO cookie generates a unique ID for every visitor                                                                                                                                                                                                  | 10 years                                        |
+| `_vwo_ds`                                     | This VWO cookie stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |
+| `_vwo_sn`                                     | This VWO cookie stores session-level information                                                                                                                                                                                                         | 30 mins and reset again to 30 mins on activity  |
+| `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | This VWO cookie stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |
+| `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 secs                                         |
```

Comment:
```suggestion
| `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |
```
should we be using the full word rather than the abbreviation, like we do in the section below? same for 'mins'?